src/libs/collect.c

```diff
@@ -2064,31 +2078,35 @@ static void _set_tooltip(dt_lib_collect_rule_t *d)
     /* xgettext:no-c-format */
                                 _("use `%' as wildcard\n"
     /* xgettext:no-c-format */
-                                  "use `|%' to include all sub-hierarchies (ctrl-click)\n"
+                                  "left-click to include hierarchy + sub-hierarchies (suffix `*')\n"
```

Comment:
we do not use "left-click" only "click" here.

Reply:
fixed (I thought it would be easier to understand with 'left-')